### PR TITLE
feat: add CollectorBaseInfoSection read case markups

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

Applied read mode markups of `CollectorBaseInfoSection` and composite related components.

https://github.com/cloudforet-io/console/assets/26986739/407e5c35-dba5-48d6-9c1b-1be03fff2992

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Comment by yuda110** [comment]
Using `reactive()` for state declaration is recommended for improved tracking of state values and readability.
But it is not mandatory.
Using `computed` or `ref` independently is allowed as needed.

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue [hunk 1]
```diff
@@ -75,12 +76,13 @@ const fields = computed<DefinitionField[]>(() => [
...
 
-const plugins = computed<PluginReferenceMap>(() => store.getters['reference/pluginItems']);
-const pluginName = computed<string>(() => (collector.value ? plugins.value[collector.value.plugin_info.plugin_id]?.label ?? '' : ''));
-const pluginIcon = computed<string>(() => (collector.value ? plugins.value[collector.value.plugin_info.plugin_id]?.icon ?? '' : ''));
+const state = reactive({
+    collector: null as null|CollectorModel,
+    loading: true,
+    pluginName: computed<string>(() => (state.collector ? plugins.value[state.collector.plugin_info.plugin_id]?.label ?? '' : '')),
+    pluginIcon: computed<string>(() => (state.collector ? plugins.value[state.collector.plugin_info.plugin_id]?.icon ?? '' : '')),
+});
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:85]: after

**Review by sulmoJ — APPROVED** [review]
LGTM!
It is so good to see that the use of annotations became active.

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:85]: I like it.

Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:85]: image

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**